UI/ClientUI.h

```diff
@@ -250,6 +255,8 @@ class ClientUI {
     /// \p m_ship_designs stores information the client knows about ship designs and their ordering.
     std::unique_ptr<ShipDesignManager> m_ship_designs;
 
+    mutable std::stringstream               m_date_format_sstream;  //!< String stream to format messages' timestamp in chat.
```

Comment:
why is this a member variable and not a function local?

Reply:
It's to re-use same stream with same format settings between calls.

Reply:
not necessary

Reply:
Fixed in base PR.